functions/export-to-s3/index.py

```diff
@@ -37,8 +37,8 @@ def handler(event, context):
             matchSnapshotRegEx = "^rds:" + db + "-\d{4}-\d{2}-\d{2}-\d{2}-\d{2}$"
             if re.match(matchSnapshotRegEx, sourceId):
                 exportTaskId = ((sourceId[4:] + '-').replace("--", "-") + messageId)[:60]
-                if exportTaskId[59] == "-":
-                    exportTaskId = exportTaskId[:59]
+                if exportTaskId[-1] == "-":
+                    exportTaskId = exportTaskId[:-1]
```

Comment:
@skwashd Hi! The docs don't mention a limit on the ExportTaskIdentifier but I think I bumped into one a long time ago when I tested this. Not sure if that changed though. Have you tried?

Reply:
@diego-ojeda-binbash I'm sure there is a maximum length of the string, but that's not the problem I hit. The issue is that the the logic assumed a fixed length of the string. If the string is shorter it breaks.